q2_boots/tests/test_core_metrics.py

```diff
@@ -192,3 +199,6 @@ def test_core_metrics_phylogenetic(self):
             output[2]['unweighted_unifrac'].view(skbio.DistanceMatrix)
         self.assertEqual(observed_unweighted_unifrac,
                          expected_unweighted_unifrac)
+
+        # vizard scatter plot returned
+        self.assertEqual(output[5].type, Visualization)
```

Comment:
Just to confirm here you are testing this is a vizard scatterplot because its the only non-collection visualization?

Reply:
Yep, exactly.